Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -110,3 +136,20 @@ public extension UIConfig {
         public var imageGalleryInteritemSpacing: CGFloat = 2
     }
 }
+
+// MARK: - Steam constants
+
+private extension UIColor {
```

Comment:
This constants will be used only inside `UIConfig.ColorPallet` and I don't this user will be benefit from them. 
Thats why it's private and inside `UIConfig` file